tests/unit/s2n_key_update_test.c

```diff
@@ -223,7 +223,7 @@ int main(int argc, char **argv)
             EXPECT_SUCCESS(s2n_stuffer_growable_alloc(&stuffer, 0));
             EXPECT_SUCCESS(s2n_connection_set_io_stuffers(&stuffer, &stuffer, client_conn));
 
-            client_conn->key_update_pending = true;
+            client_conn->key_update_pending = 1;
```

Comment:
nit: Is this change needed? I tend to prefer bools for flags as it indicates the intent better, rather than it being just some arbitrary number.